src/helpers.c

```diff
@@ -144,18 +144,18 @@ int create_required_dirs(void)
 	int ret = 0;
 	int i;
 	char *dir;
-#define STATE_DIR_COUNT 3
+#define state_dir_COUNT 3
```

Comment:
Since this one is a local macro, I would prefer that it remain named STATE_DIR_COUNT